### PR TITLE
RasterDownloader: Remove properties path parser

### LINKED ISSUE #125: WMS Time Capabilities: adapt timerange
Currently the time range for the WMS Time ends at `2021-06-29` - see [Capabilities](https://sauber-sdi.meggsimum.de/geoserver/image_mosaics/wms?service=WMS&version=1.3.0&request=GetCapabilities).

```xml
<Dimension name="time" default="2021-06-29T23:00:00Z" units="ISO8601" nearestValue="1">
2020-01-01T00:00:00.000Z/2021-06-29T23:00:00.000Z/PT1H
</Dimension>
```

The time range should be extended to the last valid time.

### PR DESCRIPTION
**Pull request by jdk24**
Edit: Revert search for properties path as this is no longer needed.

Closes #125

### CODE REVIEW

File: raster_download/src/main/java/de/meggsimum/sauber/sdi/RasterDownloader.java [hunk 1]
```diff
@@ -446,10 +446,16 @@ private void insertRaster(String fileName, String absPath, Boolean rasterExists)
 	public static String getPropertiesPath(String rasterDir) throws IOException {
 
 		File propsRootDir = new File("/opt/geoserver_data/coverages/");  // root dir of geoserver imgMosaic. Must be mapped as volume by container!
-		File rasterFile = new File(rasterDir); // extract path from downloaded raster file
-		String rasterRootDir = rasterFile.getAbsoluteFile().getParent();
 		String propPath = new String(); // Path to specific mosaic to be found 
+		
+		if (!propsRootDir.exists()) { 		// if this is the first raster ever received, the GeoServer coverages folder might be uninitiated
+			propPath = "0";
```
Comment (chrismayer) [inline raster_download/src/main/java/de/meggsimum/sauber/sdi/RasterDownloader.java:452]: No need for `"0"` here. Returning `null` would be sufficient her IMHO.

Comment (jdk24) [reply raster_download/src/main/java/de/meggsimum/sauber/sdi/RasterDownloader.java:452]: The propPath is returned to [here](https://github.com/meggsimum/sauber-sdi-docker/blob/bb479d80bcbe77c4c91489ee37414fee1a481059/raster_download/src/main/java/de/meggsimum/sauber/sdi/RasterDownloader.java#L480) and will be inserted as `null` into the `station_data.raster_metadata` table [here](https://github.com/meggsimum/sauber-sdi-docker/blob/bb479d80bcbe77c4c91489ee37414fee1a481059/raster_download/src/main/java/de/meggsimum/sauber/sdi/RasterDownloader.java#L485).

Would this not insert `null` on the dir not existing, and `0` on the dir existing but no relevant CovStore? If so I'm happy to change this if this is intended.

**Comment by chrismayer** [comment]
It think the detection of the properties path is not necessary anymore in this place. Furthermore it should be done (if necessary at all) in the `geoserver_publisher` script when the ImageMosaic store is created. So I'd vote for either:

  - closing this PR and remove the `getPropertiesPath` in a separate PR or
  - Rework this PR and remove the `getPropertiesPath`
 
Sorry for the confusion with this whole `properties_path` thing ....

**Comment by jdk24** [comment]
Removed parser from script, leave raster metadata as is, insert nothing = NULL until anything changes. Renamed PR for new purpose.

**Review by chrismayer — APPROVED** [review]
Thanks...LGTM, please merge.

**Pull request merged**